package.json

```diff
@@ -9,6 +9,7 @@
     "test:ui": "yarn lint && vitest --coverage --ui"
   },
   "dependencies": {
+    "@actions/core": "^1.9.0",
```

Comment:
@tony19 

You added a dependency, so please update `yarn.lock` as well by running `yarn install`.

Reply:
All set!